### ISSUE TITLE
Support MFA for Tesla Login

### ISSUE THREAD

**Issue by jeremypoulter** [OpenEVSE/openevse_esp32_firmware#217]
Currently MFA is not supported, needs adding, but need a test account with MFA enabled, in the short term #215  could be used to enter tokens directly

**Comment by glynhudson** [comment]
Since the Tesla integration use an unofficial Tesla API I think it will always have issues and be a pain to manage. For this reason Home Assistant has removed integrated Tesla support, I think we should do the same rather than spend dev time constantly fixing it.

There are open-source projects dedicated to Tesla integration which work very well e.g TeslaMate which makes the SoC available via MQTT ready for integration with OpenEVSE e.g 

`teslamate/cars/$car_id/battery_level`

https://docs.teslamate.org/docs/integrations/mqtt/

TeslaMate has also perfected using the streaming API to avoid waking up the vehicle 

TLDR: I vote for removing Tesla API integration and just supporting MQTT / HTTP SoC integration. What do you think @chris1howell
